config/kubernetes/sig-contributor-experience/teams.yaml

```diff
@@ -48,6 +48,25 @@ teams:
     - Phillels
     - tpepper
     privacy: closed
+  contributor-comms:
+    description: alert members of new issues with contributor comms label
+    maintainers:
+    - ameukam
+    - cblecker
+    - idvoretskyi
+    - mrbobbytables
+    - nikhita
+    - palnabarun
```

Comment:
```suggestion
```

This section is not required at all, and these folks shouldn't be pinged on this group.

Reply:
removed